### ISSUE TITLE
component script imports don't work with symlinks

### ISSUE THREAD

**Issue by TwitchBronBron** [rokucommunity/brighterscript#41]
![image](https://user-images.githubusercontent.com/2544493/78323178-0d9cf100-753f-11ea-9de6-c705ee7d26c3.png)

**Comment by jfrux** [comment]
I'm honestly having trouble getting them to work with `extends:` files in `bsconfig.json`
We have like a `roku-core` repo that I clone in and is gitignored.
I added a bsconfig.json to that repo and I'm extending it with my main repo's bsconfig.json.

It specifies in the readme that it will resolve relative paths if I read that correctly...
But it's not seeming to find it no matter what I do.

**Comment by TwitchBronBron** [comment]
Fixed by 1c998cc

**Comment by TwitchBronBron** [comment]
@jfrux this sounds like a separate issue. The `bsconfig.json` files are joined together using the javascript `Object.assign()` function, which completely replaces parent properties with child properties (as proven by [this](https://github.com/rokucommunity/brighterscript/blob/9ef4a9232f526d47bfcfce38dbfedaddaa97c672/src/util.spec.ts#L162-L178) unit test).

This works exactly the same way as typescript handles their `extends` attribute. https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#configuration-inheritance-with-extends
>files, include and exclude from the inheriting config file overwrite those from the base config file.

So TLDR, your "issue" is by design. You can't merge two different sets of files arrays at this point in time. If that's something you want, please open a separate GitHub issue and we can discuss possible solutions.
